workshop/domain-setup.ftd

```diff
@@ -0,0 +1,280 @@
+{$ds.greenmarker: Update README.md} workflow ran successfully.
+
+
+-- ds.image: Figure 5: Creating GitHub repository using fastn-template
+src: $fastn-assets.files.images.blog.figure-5.png
+width: fill-container
+
+
+-- ds.markdown:
+
+- Click on `Settings` tab and follow `Step-1` to `Step-5` shown in below image:
+
+
+-- ds.image: Figure 6: On ⚙️ `Settings` tab follow Step-1 to Step-5 
+src: $fastn-assets.files.images.blog.figure-6.png
+width: fill-container
+
+
+-- ds.markdown:
+
+- Click on `Actions` tab and check for green tick, if `pages build and 
+deployment` workflow ran successfully.
+
+
+
+-- ds.image: Figure 7: {$ds.greenmarker: pages build and deployment} workflow should run successfully
+src: $fastn-assets.files.images.blog.figure-7.png
+width: fill-container
+
+
+-- ds.markdown:
+
+Once {$ds.greenmarker: pages build and deployment} ran successfully. Your first 
+`FTD` website is live.
+
+To see your live website click on `Settings` tab and follow below steps:
+
+-- ds.image: Figure 8: On  ⚙️ {$ds.greenmarker:Settings} → {$ds.greenmarker:Pages} → click on {$ds.greenmarker: Visit site} button
+src: $fastn-assets.files.images.blog.figure-8.png
+width: fill-container
+
+
+-- ds.h2: 3: GitHub repository link to Godaddy website 
+
+We have already registered your domain using Godaddy.com. Let’s link your GitHub
+ repository with your website by adding DNS(Domain Name System) configurations 
+to it.
+
+Follow below steps to add DNS configurations:
+
+- Visit GitHub documentation to update DNS configurations. [Click here to open](https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain).
+
+- Goto Step No. 5 - Keep this documentation open in browser tab
+
+- Switch to new tab OR goto Godaddy.com account dashboard and follow below steps:
+
+- Click on your user name and select {$ds.greenmarker: My Products} link from the dropdown menu.
+
+-- ds.image: Figure 9: Click on DNS button
+src: $fastn-assets.files.images.blog.figure-9.png
+width: fill-container
+
+
+-- ds.markdown:
+
+Or
+
+- Change your .com domain name from below url: https://dcc.godaddy.com/manage/{$ds.greenmarker: jonhywalker.com}/dns
+
+And open it in browser tab you should see below screen:
+
+
+-- ds.image: Figure 10: Click on Add button
+src: $fastn-assets.files.images.blog.figure-10.png
+width: fill-container
+
+
+-- ds.markdown:
+
+- Click on Add button below box will appear on screen. Click on Type and select A 
+from the dropdown menu. As shown in Step 1 and Step 2
+
+
+-- ds.image: Figure 11: Follow Step 1 and Step 2
+src: $fastn-assets.files.images.blog.figure-11.png
+width: fill-container
+
+
+-- ds.markdown:
+
+- In Name field type @ 
+
+- In Value enter 185.199.108.153 ip address([refer Step No.2 from this document](https://docs.google.com/document/d/1aD_nol3-4QSQW4d5Tn9BgpwbDj58eQAMtbiBtZCB5I8/edit#bookmark=id.ohc2t8qa8z6x))
```

Comment:
Link to this instead: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain